MANUAL.md

```diff
@@ -0,0 +1,647 @@
+1. Select several incidents in the _Incidents table_ and press `RE-OPEN SELECTED` in the _table toolbar_.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 08.18.39.png"/>
+    
+2. Confirm re-opening.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 07.40.51.png"/>
+
+#### Close (resolve) incidents
+1. Select several incidents in the _Incidents table_ and press `CLOSE SELECTED` in the _table toolbar_.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 08.19.15.png"/>
+    
+2. Press `CLOSE NOW`. Note that you can optionally provide a closing comment if needed.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 07.46.05.png"/>
+
+#### Add acknowledgement to incidents
+1. Select several incidents in the _Incidents table_ and press `ACK` in the _table toolbar_.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 08.19.15 copy.png"/>
+    
+2. Press `SUBMIT`. Note that you can optionally provide an acknowledgement comment and/or date when these acknowledgements are no longer relevant.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 07.46.26.png"/>
+
+#### Add ticket URL to incidents
+1. Select several incidents in the _Incidents table_ and press `ADD TICKET` in the _table toolbar_.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 08.20.10.png"/>
+    
+2. Type in/paste in ticket URL into the _Valid ticket URL field_ and press `SUBMIT`. Note that URL has to be absolute (full website address).
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 08.20.59.png"/>
+    
+
+#### Remove ticket URL from incidents
+1. Select several incidents in the _Incidents table_ and press `ADD TICKET` in the _table toolbar_.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 08.20.32.png"/>
+
+2. Leave the _Valid ticket URL field_ empty and press `SUBMIT`.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-12 at 08.20.59.png"/>
+
+
+
+
+## Customize alarm notifications in Argus
+Choose **when**, **where** and **what** alarm notifications you want to receive by creating, editing and deleting _notification profiles_.
+
+### Access your notification profiles
+1. Press `PROFILES` in the header.
+
+   <img src="public/screenshots/manual/Screenshot 2022-09-08 at 13.53.49 2.png"/>
+
+### Add new notification profile
+1. Select timeslot for when to receive notifications in _Timeslot drop-down_. If drop-down menu is empty, [create a timeslot](#add-new-timeslot) first.
+2. Select what alarms you want to receive notifications about in _Filter drop-down_. If drop-down menu is empty, [create a filter first](#save-current-filter).
+3. Choose whether you want to receive notifications via SMS or Email in _Media drop-down_.
+4. Select what phone number(s) you want to receive notifications to in _Phone number drop-down_. If drop-down menu is empty, [add a phone number](#add-new-phone-number-in-settings) first.
```

Comment:
The whole section about notification profiles is based on the _old_ version of notification profiles, and will be updated when we roll-out v.2 in production. Optional phone numbers is the first PR that is marked with _API v.2_ label. So I don't think that mentioning it here is relevant since in API v.1 SMS is the only notification type. Makes no sense to create notification without notification medium, or? 🤷‍♀️

Reply:
But Email is also a notification medium
Now that I think about it it would be good to let the user know that if they select email the email will be sent to their email address.
Is there a way for the user to find out which email address they are registered with?

Reply:
But I agree with not mentioning yet that phone numbers are optional

Reply:
Is it a notification medium in API v.1?